ACID/acid_classic.sql

```diff
@@ -8981,6 +8981,7 @@ INSERT INTO `creature_ai_scripts` (`id`,`creature_id`,`event_type`,`event_invers
 -- Drulzegar Skraghook (12340) - NSR
 -- Enraged Reef Crawler
 ('1234701','12347','2','0','100','1025','20','0','120000','120000','0','0','11','8599','0','0','1','1191','0','0','0','0','0','0','Enraged Reef Crawler - Cast Enrage at 20% HP'),
+('1234702','12347','1','2','100','1','60000','120000','60000','120000','0','0','53','19','0','0','0','0','0','0','0','0','0','0','Enraged Reef Crawler - Start Relay Script on OOC'),
```

Comment:
why phasemask 2?

Reply:
I tried to fix this using event phases first, I don't remember exactly for what, but I found a better way to do it and forgot to remove it from ai. Will remove it, thanks!